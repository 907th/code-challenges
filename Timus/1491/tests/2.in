7
1 7 1
2 3 4
3 5 3
1 2 1
5 7 4
2 4 10
3 4 2
1 6 3
