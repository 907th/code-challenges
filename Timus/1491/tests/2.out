5 19 23 19 11 8 5
